### PR TITLE
Small fixes on model loading

### PR DESCRIPTION
**Pull request by vkakerbeck**
I added the random ID to be specified in the load parameter so now it is not a boolean anymore but a string of the full run name. I also added the ID into the wandb name so it's easy to associate the wandb run with the trained model folder. 

One thing we could think about is to add another flag to not save models (or not save when in debugging mode) so we don't create so many folders while debugging. Previously they were just overwritten but now with the random ID it quickly gets messy. What do you think?

I also added precomputed statistics files for the robot arm and atari breakout so it doesn't need to be calculated again.

### CODE REVIEW

**Comment by lucasosouza** [comment]
> One thing we could think about is to add another flag to not save models (or not save when in debugging mode) so we don't create so many folders while debugging. Previously they were just overwritten but now with the random ID it quickly gets messy. What do you think?

Agree, that is probably a good idea. At least on debugging mode it shouldn't save, and an additional flag as well can be useful if you think it will help.

File: projects/disagreement/run.py [hunk 1]
```diff
@@ -195,7 +199,7 @@ def make_env_all_params(rank, args):
         # TODO: Resume wandb logging is not working
         run = wandb.init(
             project="embodiedAI",
-            name=run_args.exp_name,
+            name=run_args.exp_name + logging_args.project_id,
```
Comment (lucasosouza) [inline projects/disagreement/run.py:202]: To follow the pattern of the checkpoint above, shouldn't we also add an underscore between exp_name and id? Or conversely remove the underscore from the previous. Any of those two would make them consistent.

**Review by lucasosouza — CHANGES_REQUESTED** [review]
Left a small comment on making the naming between wandb and local checkpointing consistent

File: projects/disagreement/run.py (hunk 1 above)
Comment (vkakerbeck) [reply projects/disagreement/run.py]: Right, good point, I added an underscore here.

**Pull request merged**